examples/platform/silabs/efr32/wf200/efr_spi.c

```diff
@@ -445,3 +377,102 @@ void sl_wfx_host_gpio_init(void)
     NVIC_SetPriority(GPIO_EVEN_IRQn, 5);
     NVIC_SetPriority(GPIO_ODD_IRQn, 5);
 }
+
+#if defined(EFR32MG24)
+
+void SPIDRV_ReInit(uint32_t baudrate)
+{
+    if (USART_BaudrateGet(MY_USART) == baudrate)
+    {
+        // USART synced to baudrate already
+        return;
+    }
+    // USART is used in MG24 + WF200 combination
+    USART_InitSync_TypeDef usartInit = USART_INITSYNC_DEFAULT;
+    usartInit.msbf                   = true;
+    usartInit.clockMode              = usartClockMode0;
+    usartInit.baudrate               = baudrate;
+    uint32_t databits                = SL_SPIDRV_FRAME_LENGTH - 4U + _USART_FRAME_DATABITS_FOUR;
+    usartInit.databits               = (USART_Databits_TypeDef) databits;
+    usartInit.autoCsEnable           = true;
+
+    USART_InitSync(MY_USART, &usartInit);
+}
+
+void sl_wfx_host_spiflash_cs_assert(void)
```

Comment:
Pretty sure that I've seen this function before... Could it be duplicated ? Why ?